.github/workflows/mutation.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate Mutation Test Errors
+
+on:
+  schedule:
+    - cron: '0 13 1,15 * *' # Bi-weekly – 1PM UTC (9AM EST) on the 1st and 15th of every month
```

Comment:
The ci takes long time to execute so we should make it a cron-job to show the status periodically.

Reply:
Do we have any idea how long it takes to execute?

Reply:
It takes about 2 hours to execute on my local vm, which is 12 CPU units, 8 RAM and Nvme SSD.

Reply:
Does GitHub even allow to have such long processing time? Or is there a limit on how long something can execute? I'm thinking whether to make this a CI, or to just use it as a command before releasing new (major) versions instead

Reply:
The workflow run time limitation is 32 days. It is pretty enough to this action.
https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits

I agree with you, we can make this ci run before tagging new version instead of a cron job.